students/karl_perez/lesson03/mailroom.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+#Mailroom part 1
+import sys  # imports go at the top of the filew
+
+#donors
+donor_db = [
+    ("Krystal Perez", [50.00, 250.00]),
+    ("Eddie Lau", [32.50]),
+    ("Jimmy Jack", [200.00, 350.00, 400.00]),
+    ("Grace Cool", [120.00, 75.00]),
+    ("Adriel Molina", [45.00, 450.00]),
+    ]
+
+# menu prompt
+prompt = "\n".join(("Welcome to the Mail Room!",
+          "Please choose from below options:",
+          "1 - Send a Thank you",
+          "2 - Create a report",
+          "3 - Exit",
+          ">>> "))
+
+#segregate donors
+def list_donors():
+    for i in range(len(donor_db)):
+        print("{}".format(donor_db[i][0]))
+
+# Send a thank you
+def send_thanks():
+    print("To view current donors, type 'list' ")
+    names = input("Please provide the Full Name of a donor:")
+    if names.lower() == 'list':
+        list_donors()
+        send_thanks()
+    else:
+        amount = get_amount()
+        for donor in donor_db:
+            if donor_db[0] == names:
+                donor.append((names, amount))
+                break
+        else:
+            donor_db.append((names,[amount]))
+            email(names, amount)
```

Comment:
This is adding new donations for existing donors as new donors.  Should be summing total donations.